lib/nylas/collection.rb

```diff
@@ -17,7 +17,7 @@ def initialize(model:, api:, constraints: nil)
 
     # Instantiates a new model
     def new(**attributes)
-      model.new(attributes.merge(api: api))
+      model.new(**attributes.merge(api: api))
```

Comment:
What does this change do?

Reply:
Thanks for pointing that out, I have reverted this change. That wasn't supposed to have been committed.